modules/bmo-performing-a-live-update-to-the-hostfirmwaresettings-resource.adoc

```diff
@@ -34,37 +35,38 @@ $ oc patch hostfirmwaresettings <hostname> --type merge -p \ <1>
 +
 [NOTE]
 ====
-Get the `FirmwareSchema` resource to determine which settings the hardware supports. You can also use `oc edit <hostname> hostfirmwaresettings -n openshift-machine-api` to update the resource.
+Get the `FirmwareSchema` resource to determine which settings the hardware supports and what settings and values you can update. You cannot update read-only values and you cannot update the `FirmwareSchema` resource. You can also use `oc edit <hostname> hostfirmwaresettings -n openshift-machine-api` to update the resource.
 ====
 
-. Cordon the host by running the following command:
+. Cordon the node by running the following command:
 +
 [source,terminal]
 ----
-$ oc cordon <hostname>
+$ oc cordon <node_name> <1>
 ----
-+
-The Bare Metal Operator (BMO) updates the `operationalStatus` parameter to `servicing`.
+<1> Replace `<node_name>` with the name of the node. The Bare Metal Operator (BMO) updates the `operationalStatus` parameter to `servicing`.
```

Comment:
<1> Replace `<node_name>` with the name of the node. The Bare Metal Operator (BMO) updates the `Operational Status` of the associated `BareMetalHost` parameter to `servicing`.
(clarifying Operational Status is a BMH property not node property)